ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/EVPN_MULTICAST_TESTS.yml

```
@@ -20,6 +20,10 @@ sflow_settings:
 fabric_sflow:
   l3_interfaces: true
 
+bgp_peer_groups:
+  mlag_ipv4_vrfs_peer:
+    name: MLAG-IPv4-OVERLAY-PEER
```

same please change the name
```suggestion
    name: MLAG-IPv4-VRFS-PEER
```

---

Done!